app/services/db/doc_auth_log/doc_auth_funnel_summary_stats.rb

```diff
@@ -0,0 +1,57 @@
+module Db
+  module DocAuthLog
+    class DocAuthFunnelSummaryStats
+      SKIP_FIELDS = %w[id user_id].freeze
+
+      def call
+        total_count = ::DocAuthLog.count
+        return {} if total_count.zero?
+        convert_percentages(total_count)
+        results['total_verified_users_count'] = ::DocAuthLog.verified_users_count
```

Comment:
thought about separating this into summary subhash and others into detailed subhash